packages/medusa/src/repositories/image.ts

```diff
@@ -1,8 +1,25 @@
 import { EntityRepository, In, Repository } from "typeorm"
-import { Image } from "../models/image"
+import { Image } from "../models"
+import { QueryDeepPartialEntity } from "typeorm/query-builder/QueryPartialEntity"
 
 @EntityRepository(Image)
 export class ImageRepository extends Repository<Image> {
+  async insertBulk(data: QueryDeepPartialEntity<Image>[]): Promise<Image[]> {
```

Comment:
`insertBulk` would imply that this is doing a bulk insert, but this looks just like an createMany instead